staff/jaume-serradell/gamma/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Gamma</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <p class="navigation__logo">Gamma</p>
+            <ul class="navigation__list">
+                <li class="navigation__list-items"><a class="navigation__list-items-anchors" href="#">Home</a></li>
```

Comment:
For BEM would be better if you write "navigation__list-items" =>  "navigation__list__items" 
I am not sure if it works as you did, but as i have understood it should with "x__x" ;D